### ISSUE TITLE
[Bullet] When using _integrate_forces() the PhysicsDirectBodyState.total_gravity returns (0,0,0)

### ISSUE THREAD

**Issue by kubecz3k** [godotengine/godot#65773]
### Godot version

5600be95713dff993160777161ae30b695fb60b9

### System information

Ubuntu 22.04, 6700xt

### Issue description

In 3.5 every body that is using _integrate_forces() don't have access to correct gravity information, 0 vector is always provided
This is a regression it was behaving as intended both in 3.3.2 as well as in 3.4.5, it also behaves like intended with Godot physics

### Steps to reproduce

0. Create new project with default settings
1. Create new scene with a rigidbody
2. Setup rigidbody: add collision, mark `custom_integrator` checkbox
3. Add a script to a rigidbody:
```
func _integrate_forces(state: PhysicsDirectBodyState) -> void:
	print(state.get_total_gravity())
```
4. Run the scene, observe (0,0,0) vector is being printed

### Minimal reproduction project

[IntegrationGravity.zip](https://github.com/godotengine/godot/files/9563421/IntegrationGravity.zip)

**Comment by akien-mga** [comment]
Indeed this was changed by @madmiraal in #62708.

The documentation likely needs to make it clear as it's not obvious that this will happen.

**Comment by kubecz3k** [comment]
@akien-mga  Yes but I would argue it's not the right fix, there is no good way to obtain gravity information in bullet anymore (still works in Godot physics). The workaround is not as easy as just taking the value from Project settings since you might have (and I have) areas which are influencing gravity in the game.

**Comment by kubecz3k** [comment]
I think there might be some misunderstand here, To make this clear, it's correct behavior that gravity is not applied automatically to bodies which are using manual integration. What's not correct is that those bodies cannot apply that gravity themselves correctly anymore since engine is not providing gravity information anymore

**Comment by akien-mga** [comment]
Fixed by #69823.
